src/d3a/models/myco_matcher/external_matcher.py

```diff
@@ -37,16 +36,16 @@ def _setup_redis_connection(self):
         self.myco_ext_conn = ResettableCommunicator()
         self.myco_ext_conn.sub_to_multiple_channels(
             {"external-myco/simulation-id/": self.publish_simulation_id,
-             f"{self._channel_prefix}/offers-bids/": self.publish_offers_bids,
+             f"{self._channel_prefix}/matching-data/": self.publish_matching_data,
```

Comment:
I don't see any PR that changes this on the `myco-api-client`. Is there something missing?

Reply:
Yes, the myco PR is still not created